src/components/content-tab-snapshots.tsx

```diff
@@ -435,21 +457,36 @@ function AddDemoSiteWithProgress( {
 					</div>
 				</div>
 			) : (
-				<Tooltip disabled={ ! tooltipContent } { ...tooltipContent }>
-					<Button
-						aria-description={ tooltipContent?.text ?? '' }
-						aria-disabled={ isDisabled }
-						variant="primary"
-						onClick={ () => {
-							if ( isDisabled ) {
-								return;
-							}
-							archiveSite( selectedSite.id );
-						} }
-					>
-						{ __( 'Add demo site' ) }
-					</Button>
-				</Tooltip>
+				<div className="flex gap-4">
+					<Tooltip disabled={ ! addTooltipContent } { ...addTooltipContent }>
+						<Button
+							aria-description={ addTooltipContent?.text ?? '' }
+							aria-disabled={ isDisabled }
+							variant="primary"
+							onClick={ () => {
+								if ( isDisabled ) {
+									return;
+								}
+								archiveSite( selectedSite.id );
+							} }
+						>
+							{ __( 'Add demo site' ) }
+						</Button>
+					</Tooltip>
+					{ isExpired && (
+						<Tooltip disabled={ ! clearTooltipContent } { ...clearTooltipContent }>
+							<Button
+								aria-description={ clearTooltipContent?.text ?? '' }
+								disabled={ isOffline }
```

Comment:
As it's about deleting a site from the local storage only, do we need to disable it in offline mode?

Reply:
The reason why I initially implemented it was because I re-used `deleteSnapshot` and it first makes an API request and then removes the snapshots stored locally so it would fail in the offline mode. Also, it seemed consistent in the UI with adding a demo site when the site was offline.

 On a second thought, I could probably use `removeSnapshot` directly instead and only target the locally stored snapshots in `appdata-v1.json`.

I will re-test now with the approach you mentioned for testing.